path_login.go

```diff
@@ -102,8 +104,14 @@ func (b *kubeAuthBackend) pathLogin(ctx context.Context, req *logical.Request, d
 		return nil, err
 	}
 
+	// ensure the http client is not empty
+	if b.httpClient == nil {
+		b.httpClient = cleanhttp.DefaultPooledClient()
+	}
+
```

Comment:
I don't think this is necessary. We should never instantiate the plugin without using the `Backend` function

Reply:
That makes sense! Changed in [dac4c83](https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/142/commits/dac4c8339f2b90997f7148f3832e0df1ad5ae525)